### DISCUSSION TITLE
Is there a way to show all candidates for certain actions (e.g. buffer list)?

### DISCUSSION THREAD

**Discussion by ZedThree** [emacs-helm/helm#2624] [Q&A] [answered]
`C-x b` to show the list of buffers is limited to `helm-candidate-number-limit`, and this is before the "boring buffers" are removed. Is there a nice way to show all open buffers (with/without the boring ones)?

**Comment by thierryvolpiatto** [comment, marked as answer]
Peter Hill ***@***.***> writes:

> 1.  ( ) text/plain          (*) text/html           
>
> C-x b to show the list of buffers is limited to
> helm-candidate-number-limit, and this is before the "boring buffers"
> are removed. Is there a nice way to show all open buffers
> (with/without the boring ones)?

Don't know to what `C-x b` is bound.
But with M-x helm-buffers-list or M-x helm-mini, you can use `C-c a`.
Oh and I see this not documented in C-h m, will add this soon.
Thanks.

> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.*Message ID: ***@***.***>


-- 
Thierry

**Reply by ZedThree** [reply]
Perfect, thanks!
